e2e/starter.test.js

```diff
@@ -0,0 +1,19 @@
+describe('Example', () => {
```

Comment:
Why is this a plain JavaScript file? Use TypeScript throughout.

Reply:
Updated to TypeScript